File: BackEnd/Zalo_mini/Zalo_mini/Models/call_participant.cs
using System;
using System.Collections.Generic;

namespace Zalo_mini.Models
{
    public partial class call_participant
    {
        public long id { get; set; }
        public long call_id { get; set; }
        public long user_id { get; set; }
        public DateTime? joined_at { get; set; }
        public DateTime? left_at { get; set; }
        public string? status { get; set; }

        public virtual call call { get; set; } = null!;
        public virtual user user { get; set; } = null!;
    }
}

File: BackEnd/java/ZaloMini/src/main/java/com/devmam/zalomini/entities/CallParticipant.java
package com.devmam.zalomini.entities;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.ColumnDefault;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.OnDelete;
import org.hibernate.annotations.OnDeleteAction;

import java.time.Instant;
import java.util.Date;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity(name = "CallParticipant")
@Table(name = "call_participants")
public class CallParticipant {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @OnDelete(action = OnDeleteAction.CASCADE)
    @JoinColumn(name = "call_id", nullable = false)
    private Call call;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @OnDelete(action = OnDeleteAction.CASCADE)
    @JoinColumn(name = "user_id", nullable = false)
    private User user;

    @Column(name = "joined_at")
    @CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)
    private Date joinedAt = new Date();

    @Column(name = "left_at")
    @CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)
    private Date leftAt;

    @ColumnDefault("'invited'")
    @Lob
    @Column(name = "status")
    private String status;

}